src/dso_api/lib/schematools/models.py

```diff
@@ -114,67 +129,60 @@ def fetch_srid(dataset: DatasetSchema, field: DatasetFieldSchema) -> Dict[str, A
 
 
 JSON_TYPE_TO_DJANGO = {
-    "string": FieldMaker(UnlimitedCharField),
-    "integer": FieldMaker(models.IntegerField),
-    "number": FieldMaker(models.FloatField),
-    "boolean": FieldMaker(models.BooleanField),
-    "/definitions/id": FieldMaker(models.IntegerField),
-    "/definitions/schema": FieldMaker(UnlimitedCharField),
-    "https://geojson.org/schema/Geometry.json": FieldMaker(
+    "string": (UnlimitedCharField, None),
+    "integer": (models.IntegerField, None),
+    "number": (models.FloatField, None),
+    "boolean": (models.BooleanField, None),
+    "array": (ArrayField, None),
+    "/definitions/id": (models.IntegerField, None),
+    "/definitions/schema": (UnlimitedCharField, None),
+    "https://geojson.org/schema/Geometry.json": (
```

Comment:
Wat is het voordeel van `FieldMaker` er hier weer uithalen?

Reply:
Ik had `FieldMaker` uitgehaald, omdat `JSON_TO_FIELD` mapping kan in andere plekken gebruikt worden.